sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1186,6 +1186,80 @@ def max_deflection(self):
         else:
             return None
 
+
+    def bending_stress(self):
```

Comment:
seems like this should be `bending_stress` and have it not return an expression with `y` (which is something you should not do in general to avoid clashing with any other symbol that might be present). Geometry has to deal with this a lot; ode solver, too.

Reply:
Should we be using dummy variables in this case to avoid clashing?